app/models/prescription.rb

```diff
@@ -3,17 +3,17 @@
 # Prescription model
 # Reference: https://github.com/department-of-veterans-affairs/prescriptions-team/blob/master/API/sample_mvh_api_calls
 class Prescription < Common::Base
-  attribute :prescription_id, Integer
-  attribute :refill_status, String
+  attribute :prescription_id, Integer, sortable: { order: 'ASC' }, filterable: %w(eq not_eq)
+  attribute :refill_status, String, sortable: { order: 'ASC' }, filterable: %w(eq not_eq)
   attribute :refill_submit_date, Common::UTCTime
-  attribute :refill_date, Common::UTCTime
+  attribute :refill_date, Common::UTCTime, sortable: { order: 'DESC' }
   attribute :refill_remaining, Integer
-  attribute :facility_name, String
-  attribute :ordered_date, Common::UTCTime
+  attribute :facility_name, String, sortable: { order: 'ASC' }, filterable: %w(eq not_eq)
+  attribute :ordered_date, Common::UTCTime, sortable: { order: 'DESC', default: true }
   attribute :quantity, Integer
   attribute :expiration_date, Common::UTCTime
   attribute :prescription_number, String
-  attribute :prescription_name, String
+  attribute :prescription_name, String, sortable: { order: 'ASC' }
```

Comment:
I agree it would be useful, but this was mostly meant to be a refactor. I didn't not intend on modifying existing business logic.